surefire-booter/src/test/java/org/apache/maven/surefire/booter/ForkedBooterMockTest.java

```diff
@@ -385,4 +472,26 @@ public Object answer( InvocationOnMock invocation )
         } ).when( ShutdownHookUtils.class, "addShutDownHook", any( Thread.class ) );
         invokeMethod( booter, "flushEventChannelOnExit" );
     }
+
+    @Test
+    public void shouldParseUUID() throws Exception
+    {
+        SurefireMasterProcessChannelProcessorFactory factory = new SurefireMasterProcessChannelProcessorFactory();
+        UUID uuid = UUID.randomUUID();
+        URI uri = new URI( "tcp://127.0.0.1:12345?auth=" + uuid );
```

Comment:
Still see IP address here.

Reply:
This needs to be changed in current branch.

Reply:
@michael-o 
Done!